README.md

```diff
@@ -13,16 +13,26 @@ issues, mailing lists, conferences, etc.
 
 For more specific topics, try a SIG.
 
-## SIGs
+## Governance
 
-Kubernetes is a set of subprojects, each shepherded by a Special Interest Group (SIG).
+Kubernetes has three types of groups that are officially supported:
 
-A first step to contributing is to pick from the [list of kubernetes SIGs](sig-list.md).
+* **Committees** are named sets of people that are chartered to take on sensitive topics.
+  This group is encouraged to be as open as possible while achieving its mission.
+  Examples of committees include the steering committee and things like security or code of conduct.
+* **Special Interest Groups (SIGs)** are persistent open groups that focus on a part of the project.
```

Comment:
what does it mean to be open in membership?

Reply:
transparent?

Reply:
> > what does it mean to be open in membership?
>
> transparent?

An alternative interpretation would be that that self-nomination is sufficient to join (i.e. that there's no new-member vetting process).  Perhaps the wording here (and/or in `sig-governance.md`?) could be updated to make it clear which of those is intended?  Do non-lead SIG members have any explicit powers?

Reply:
Right now in the steering committee we are working on guidance/process for the level of "membership" needed to participate in processes like voting for the steering committee.  As part of that I don't want to overload the term "membership".  I'll reword this to say that anyone is welcome to contribute and participate.